lib/matplotlib/backends/backend_ps.py

```diff
@@ -802,7 +802,7 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
                 else:
                     kern = 0
                 lastgind = gind
-                thisx += kern/64.0
+                thisx -= kern/64.0
```

Comment:
why did the sign of the kern adjustment change?  This no longer matches the sign in the pdf backend...